src/modules/info-display/state/info-display-slice.ts

```diff
@@ -0,0 +1,26 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+import { RootState } from '@/store'
+
+interface DisplayState {
+  message: string
+}
+
+const initialState: DisplayState = {
+  message: '',
+}
+
+const displaySlice = createSlice({
+  name: 'display',
+  initialState,
+  reducers: {
+    setMessage(state, action: PayloadAction<DisplayState>) {
+      state.message = action.payload.message
+    },
+  },
+})
+
+export const { setMessage } = displaySlice.actions
+
+export const selectDisplayMessage = (state: RootState) => state.display.message
```

Comment:
Better to create next structure
```
src
- modules
-- info-display
--- redux
---- info-display.slice.ts
---- info-display.selectors.ts
```
The reson for this is that this that we specify that it's not just state but specifically redux.